cli/reporter.py

```diff
@@ -226,9 +230,87 @@ def clean_read(ifile):
                         break  # EOF
                     errors.append({'property': line, 'description': line2})
         self.results['metadata_check'] = {
-            'report_path': metadata_check_file,
+            'json_report_path': metadata_check_file,
             'json_errors': errors
         }
+        self.convert_metadata_validation_results()
+        self.write_converted_metadata_reslts()
+
+
+
+    def _parse_metadata_property(self, property_str):
+        if property_str.startswith('.'):
+            return property_str.strip('.'), None, None
+        match = re.match(r'/(\w+)(/(\d+))?(\.(\w+))?', property_str)
+        if match:
+            return match.group(1), match.group(3), match.group(5)
+        else:
+            logger.error(f'Cannot parse {property_str} in JSON metadata error')
+            return None, None, None
+
+    def convert_metadata_validation_results(self):
```

Comment:
Judgment call: 
```suggestion
    def user_friendly_metadata_validation_results(self):
```

Reply:
I tried to make the function name clearer.